Pinta/Actions/File/NewDocumentAction.cs

```diff
@@ -73,7 +73,6 @@ private void Activated (object sender, EventArgs e)
 				PintaCore.Settings.PutSetting ("new-image-width", dialog.NewImageWidth);
 				PintaCore.Settings.PutSetting ("new-image-height", dialog.NewImageHeight);
 				PintaCore.Settings.PutSetting ("new-image-bg", dialog.NewImageBackgroundType);
-				PintaCore.Settings.SaveSettings ();
```

Comment:
I suppose one downside with not immediately saving the settings to disk is that the changes could be lost if there's a hard crash (e.g. a crash in Mono). I don't feel too strongly about it though